jsonrpsee/Cargo.toml

```diff
@@ -12,6 +12,7 @@ http-server = { path = "../http-server", version = "0.2.0-alpha.4", package = "j
 ws-client = { path = "../ws-client", version = "0.2.0-alpha.4", package = "jsonrpsee-ws-client", optional = true }
 ws-server = { path = "../ws-server", version = "0.2.0-alpha.4", package = "jsonrpsee-ws-server", optional = true }
 proc-macros = { path = "../proc-macros", version = "0.2.0-alpha.4", package = "jsonrpsee-proc-macros", optional = true }
+types = { path = "../types", version = "0.2.0-alpha.4", package = "jsonrpsee-types", optional = true }
```

Comment:
I had to include this again, if not  any of the `client crates` is in the users `Cargo.toml` it won't work.

That's the reason why I moved the tests to separate crate, is just worked before because it was in the same crate.